File: Pratica 6 - Funcoes 2/cpp_logic/exercicio6_v1.cpp
#include <iostream>
using namespace std;

int main() {
    float valor, desc;
    cout << "Digite o valor do produto: ";
    cin >> valor;
    cout << "Digite o desconto (%): ";
    cin >> desc;

    float final = valor - (valor * desc / 100);
    cout << "O valor final é: R$ " << final;

    return 0;
}

File: Pratica 6 - Funcoes 2/java_logic/Exercicio6_v1.java
import java.util.Scanner;

public class Exercicio6_v1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite o valor do produto: ");
        double valor = sc.nextDouble();
        System.out.println("Digite o valor do desconto(%): ");
        double desc = sc.nextDouble();

        double final_valor = valor - (valor * desc / 100);

        System.out.println("O valor final do produto é: " + final_valor);

        sc.close();
    }
}
